dotcom-rendering/src/types/article.ts

```diff
@@ -36,6 +37,10 @@ export const enhanceArticleType = (
 ): Article => {
 	const format = decideFormat(data.format);
 
+	const crosswordBlock = buildCrosswordBlock(data);
+
+	const additionalBlocks = crosswordBlock ? [crosswordBlock] : [];
```

Comment:
To keep all 'block' related functionality in one place could this be moved into `enhanceBlocks` and we pass an optional `data.crosswords` here?

Reply:
We paired on this and moved `additionalBlocks` out of `enhanceBlocks` and up to `enhanceArticleType`